orekit/tests/test_twobody_orbit.py

```diff
@@ -0,0 +1,55 @@
+""" Test cases for two-body orbit behaviors """
+
+# Import required orekit data in current directory
+import pytest
+from astropy import units as u
+from astropy.tests.helper import assert_quantity_allclose
+from numpy.testing import assert_allclose
+from orekit_orbit import OrekitOrbit
+from poliastro.bodies import Earth
+from poliastro.twobody import Orbit
+from utils import setup_orekit_env
+
+# Start orekit virtual machine and load data
+vm = setup_orekit_env()
+
+
+def test_orekit_orbit_from_rv():
+
+    # Initial RV elements
+    r = [-6045, -3490, 2500] * u.km
+    v = [-3.457, 6.618, 2.533] * u.km / u.s
+
+    # Generate orekit and poliastro orbits
+    ss_orekit = OrekitOrbit.from_vectors(Earth, r, v)
+    ss_poliastro = Orbit.from_vectors(Earth, r, v)
+
+    # Assert position and velocity vectors
+    assert_quantity_allclose(ss_poliastro.a, ss_orekit.a)
+    assert_quantity_allclose(ss_poliastro.ecc, ss_orekit.ecc)
+    assert_quantity_allclose(ss_poliastro.inc, ss_orekit.inc)
+    assert_quantity_allclose(ss_poliastro.raan, ss_orekit.raan)
```

Comment:
This test is currently failing on my laptop:

```
E       AssertionError: 
E       Not equal to tolerance rtol=1e-07, atol=0
E       
E       Mismatched elements: 1 / 1 (100%)
E       Max absolute difference: 6.28318531
E       Max relative difference: 3.43771527
E        x: array(4.455464)
E        y: array(-1.827721)
```

Reply:
Orekit computes the `raan` in the following [line of code](https://gitlab.orekit.org/orekit/orekit/-/blob/develop/src/main/java/org/orekit/orbits/KeplerianOrbit.java#L321):

```java
raan = Vector3D.crossProduct(Vector3D.PLUS_K, momentum).getAlpha();
```

Previous `getAlpha()` bounds the output angle between `-pi` and `+pi`, as stated in Hipparchus (the Java math library I guess) [official docs](https://www.hipparchus.org/apidocs/org/hipparchus/geometry/euclidean/threed/Vector3D.html#getAlpha--). However, this angle is claimed to be within the range `[0, 2pi)` in Vallado book and the approach followed in poliastro `rv2coe`.

I just bounded the angle between 0 and 2pi and all the tests are passing now :+1: